lib/components/Input/base/index.tsx

```diff
@@ -28,7 +28,7 @@ export interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
     status?: Status;
     disabled?: boolean;
     variant?: Variants;
-    message?: string;
+    message?: ReactNode;
```

Comment:
```suggestion
    message?: ReactNode | string;
```

Reply:
I omitted the string since `ReactNode` represents `string` as well. is this necessary?